scipy/spatial/transform/rotation.py

```diff
@@ -1,12 +1,55 @@
 from __future__ import division, print_function, absolute_import
 
+__all__ = ['Rotation']
+
 import numpy as np
+import scipy.linalg
 
 class Rotation(object):
     """Rotation in 3 dimensions.
 
     This class will include initializers from different representations, converters
```

Comment:
Sure this documentation is preliminarily. While we are at it, I think it should explain:

1. What rotation is mathematically.
2. How do we view a rotation (as physical rotation of the original frame to the final frame).
3. As the consequence of 2 if we have the chain of rotations A -> B -> C, the R_AC = R_AB * R_BC

You don't need to focus on this documentation now, but something to keep in mind.

Reply:
I'll keep that in mind. I was planning to finalize documentation once the implementation was done.

Reply:
This is not PEP8 compliant (>80 char). I'd recommend setting up automated PEP8 checking in your editor.